test/e2e/tests/swap-send/swap-send-eth-with-loaded-state.spec.ts

```diff
@@ -0,0 +1,297 @@
+      .thenCallback(() => ({
+        statusCode: 200,
+        json: {
+          active: true,
+          networkId: 1,
+          chainId: 1,
+          chainName: 'Ethereum Mainnet',
+          nativeCurrency: {
+            name: 'Ether',
+            symbol: 'ETH',
+            decimals: 18,
+            address: '0x0000000000000000000000000000000000000000',
+          },
+          iconUrl: 'https://s3.amazonaws.com/airswap-token-images/ETH.png',
+          blockExplorerUrl: 'https://etherscan.io',
+          networkType: 'L1',
+          aggregators: [
+            'airswapV3',
+            'airswapV4',
+            'oneInchV4',
+            'oneInchV5',
+            'paraswap',
+            'pmm',
+            'zeroEx',
+            'openOcean',
+            'hashFlow',
+            'wrappedNative',
+            'kyberSwap',
+            'airSwapV4_3',
+            'hashFlowV3',
+          ],
+          refreshRates: {
+            quotes: 30,
+            quotesPrefetching: 30,
+            stxGetTransactions: 10,
+            stxBatchStatus: 1,
+            stxStatusDeadline: 160,
+            stxMaxFeeMultiplier: 2,
+          },
+          parameters: {
+            refreshRates: {
+              quotes: 30,
+              quotesPrefetching: 30,
+              stxGetTransactions: 10,
+              stxBatchStatus: 1,
+            },
+            stxStatusDeadline: 160,
+            stxMaxFeeMultiplier: 2,
+          },
+        },
+      })),
+  ];
+}
+
+describe('Swap-Send ETH', function () {
+  describe('to non-contract address with data that matches swap data signature', function (this: Suite) {
+    it.only('submits a transaction successfully with max amount', async function () {
+      await withFixtures(
+        {
+          fixtures: new FixtureBuilder()
+            .withNetworkControllerOnMainnet()
+            .withTokensController({
+              allTokens: {
+                '0x1': {
+                  '0x5cfe73b6021e818b776b421b1c4db2474086a7e1': [
+                    {
+                      address: '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2',
+                      symbol: 'WETH',
+                      decimals: 18,
+                      isERC721: false,
+                      aggregators: [],
+                    },
+                  ],
+                },
+              },
+            })
+            .build(),
+          title: this.test?.fullTitle(),
+          testSpecificMock: mockSwapQuotes,
+          localNodeOptions: [
+            {
+              type: 'anvil',
+              options: {
+                chainId: 1,
+                hardfork: 'london',
+                loadState:
+                  './test/e2e/seeder/network-states/swap-state/withSwapContracts.json',
+              },
+            },
+          ],
+        },
+        async ({ driver }) => {
+          const swapSendPage = new SwapSendPage(driver);
+          await logInWithBalanceValidation(driver);
+
```

Comment:
maybe we could check the initial token balances, so we make sure those are right at the beginning of the spec?

```suggestion
await homePage.check_expectedTokenBalanceIsDisplayed('50', 'WETH');
await homePage.check_expectedTokenBalanceIsDisplayed('25', 'ETH');

```

Reply:
this check will validate the 50ETH but not the 50 WETH. Could we add that too? I think it make sense for the test as we are swapping between the 2 assets, so we can check both balances before and after, if that makes sense?

await homePage.check_expectedTokenBalanceIsDisplayed('50', 'WETH');